java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2550,6 +2552,56 @@ public T touch(@NonNull String[] keys) {
         return getThis();
     }
 
+    /**
+     * Counts the number of set bits (population counting) in a string of specified <code>key</code>.
```

Comment:
Copy that to interface and to all methods here
```suggestion
     * Counts the number of set bits (population counting) in a string stored at <code>key</code>.
```

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00